org.eclipse.jdt.ls.tests/META-INF/MANIFEST.MF

```diff
@@ -14,6 +14,7 @@ Require-Bundle: org.eclipse.jdt.ls.core,
  org.eclipse.core.resources,
  org.eclipse.jdt.apt.core,
  org.eclipse.jdt.core,
+ org.eclipse.jdt.core.javac,
```

Comment:
This is a fragment, it cannot be added here.
You can try `Import-Package: org.eclipse.jdt.internal.javac`, but this first requires to add `org.clipse.jdt.ls.tests` as friend in o.e.j.c.javac fragment.

Reply:
thanks for the hint.
x:friend was not necessary though.